libs/designsystem/src/lib/components/list/list.component.ts

```diff
@@ -150,6 +151,7 @@ export class ListComponent implements OnInit, AfterViewInit, OnChanges {
     this.hasDeprecatedItemTemplate = !!this.legacyItemTemplate || !!this.legacyFlexItemTemplate;
     this.initializeSwipeActions();
     this.isSelectable = this.itemSelect.observers.length > 0;
+
```

Comment:
Any reason for this line break?

Reply:
Not at all 😄 - should be resolved by f311d2d.